app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :thoughts, :dependent => :destroy
+
+  def full_name
+    [first_name, last_name].join(' ')
+  end
```

Comment:
if full_name is going to be displayed on the FE, does it also need to be made pretty where the `first_name` and `last_name` need to be capitalized?

Reply:
Yeah, I guess we can add extra method to capitalize here :)

Reply:
May be write some test too :D